docs/pnpm.md

```diff
@@ -275,6 +275,8 @@ which is equivalent to setting the `lifecycle_hooks` to an empty list for that p
 
 You can set environment variables for hook build actions using the `lifecycle_hooks_envs` attribute of `npm_translate_lock`.
 
+Some hooks may depend on environment variables specified depending on [use_default_shell_env](https://bazel.build/rules/lib/builtins/actions#run.use_default_shell_env) which may be enabled for hook build actions using the `lifecycle_hooks_use_default_shell_env` attribute of `npm_translate_lock`.
```

Comment:
Worth mentioning that this feature depends on a minimum version of bazel-lib that adds `use_default_shell_env` to `run_binary. The MODULE.bazel of rules_js currently brings in 1.x bazel-lib that I'm not sure has that attribute.

Should also mention the minimum version in the docstrings of npm_translate_lock and npm_import if there is one